src/core/connectivity.jl

```diff
@@ -264,7 +338,8 @@ with_excluded_positions(
 with_excluded_positions(
     c::LatticeConnectivity,
     excluded_positions::Vector{Int},
-)::LatticeConnectivity = LatticeConnectivity(c.qubits_per_row, excluded_positions)
+)::LatticeConnectivity =
+    LatticeConnectivity(c.qubits_per_row, excluded_positions, c.excluded_connections)
```

Comment:
is gonna break after rebasing, needs:
`LatticeConnectivity(c.dimensions[1], c.dimensions[2], excluded_positions, c.excluded_connections)`